### PR TITLE
Issue 8 new readme and sample

### PR DESCRIPTION
**Pull request by dwk2**
Completely new readme and sample file; improved documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,122 +1,129 @@
...
-the documentation for drush alias files:
+* ```drush sa```                          [list all site aliases] [shortcut: "sa" = "site-alias"]
+* ```drush sa @sitespinner-sample```      [shows execution view of a specific alias file, without any actual execution]
+* ```drush topic docs-aliases```          [documentation on writing and installing drush site aliases]
+* ```drush help sa```                     [documentation for site alias]
+* ```drush help```                        [documentation for drush]
+* ```drush help sitespinner```            [documentation for sitespinner create]
+* ```drush help sitespinner-delete```     [documentation for sitespinner delete]
+* ```drush cc drush```                    [clear drush cache] [shortcut: "cc" = "cache-clear"]
+* ```drush topic docs-aliases```          [helpful information on writing and installing drush site aliases]
```
Comment (patdunlavey) [inline README.md]: This is a duplicate.

File: README.md [hunk 2]
```diff
@@ -125,35 +132,24 @@ files.
...
-will be no changes to your system.
+Contact the authors if you want to submit pull requests to us.
 
-    -f
+## Authors
 
-Force the script to continue when errors are encountered. E.g.:
-You have tried to install a site but your failed halfway due to permissions.
-You can then change the permissions on your server, and force the script to execute.
+* **Chris Warren** - *Initial version of Sitespinner in PHP*
```
Comment (patdunlavey) [inline README.md]: This is not correct. Sitespinner was originally written as bash scripts by Andy Cavenaugh, then at Common Media. I rewrote it as a drush command. Chris did not do any coding, as I recall, but of course he developed the concept and oversaw development of both versions.

File: examples/sitespinner-sample.aliases.drushrc.php [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+	#--- Preexisting template (source) multi-site ---#
+	$TEMPLATE_SITE_DIRECTORY = "templatesite";                  # directory of live template site to be copied; alphanumeric or dashes; no spaces or underscores; omit trailing slash "/"
+	#--- Server host names ---#
+	$DOMAIN_NAME          = "subdomain.domain.edu";             # subdomain.domain.edu; leave empty if none exists; omit "https://"; omit trailing slash "/"
+	$FEDORA_HOST_NAME     = "subdomain-fedora.domain.edu";      # full host name (including subdomain, if exists) of server running Fedora (fedora.domain.edu)
+	$DRUPAL_WEB_ROOT_PATH = "/var/www/html";                    # "root" is where the web server's index.php file is located, not the multi-site root; omit trailing slash "/"
+	$SECURITY_MODE        = "https://";                         # "https://" or "http://"
+	#--- MySQL authentication (consider granting all privileges for this db user: absolutely requires CREATE, SELECT, INSERT, DELETE, DROP) ---#
+	$MYSQL_HOST_NAME = "subdomain-webdb.domain.edu";            # full host name (include subdomain, if exists) of server running MySQL (mysql.domain.edu)
```
Comment (patdunlavey) [inline examples/sitespinner-sample.aliases.drushrc.php]: You really should provide two mysql users, one for the site creator, and the other for the drupal user. The user that runs the site should not have the ability to create and destroy databases.

**Comment by patdunlavey** [comment]
Hi David, Excellent work, thank you so much!! 

I found a few things that I think need to be fixed. See my comments.

**Comment by dwk2** [comment]
Hi Pat,
Thank you for your careful read. I've completed your suggested corrections:
- removed duplicate line
- corrected author information
- added a TODO item noting: "Consider providing two mysql users: one for
the site creator, and the other for the drupal user. The user that runs the
site should not have the ability to create and destroy databases."

The final one I've put on my own TODO list to complete later.

If you think this is ready, please merge.

Thank you,
David

**Comment by patdunlavey** [comment]
Hi David, I don't think I want to merge this until the db creator and db user roles are done. That's an important regression.

**Comment by dwk2** [comment]
Pat, I've completed the updates that separate the user roles and removes the regression. Please take another look and either merge or let me know if anything else remains. I've tested this code, and it works.

**Comment by patdunlavey** [comment]
README.md:
The documentation for creating the db_creator user indicates that the user's name is "db_creator", which only happens to be correct for the sample alias file provided. Also, in markdown, the \` character renders like a `<code>` tag. I think you can escape it with a backslash, like this <code>\\`</code>.

**Comment by patdunlavey** [comment]
Or better, just wrap the whole mysql command in `<code></code>` tags.

**Comment by dwk2** [comment]
Edits completed. Please merge.

**Comment by patdunlavey** [comment]
Looks good. Thanks for your work on this David, and your patience with my picky review process!

**Pull request merged**